views/jsconfig.handlebars

```diff
@@ -5,11 +5,24 @@ if (isIE && !isUnsupportedPage) {
   window.location.replace('/unsupported/ie');
 }
 
-{{#if dsn}}
-window.dsn = '{{{dsn}}}';
+{{#if sentryId}}
+var RAVEN_CONFIG = {
+  release: '{{{version}}}',
+  tags: {
+    commit: '{{{commit}}}'
+  },
+  dataCallback: function (data) {
+    var hash = window.location.hash;
+    if (hash) {
+      return JSON.parse(JSON.stringify(data).replace(new Regexp(hash.slice(1), 'g'), ''));
```

Comment:
Should be `RegExp` (per [MDN](https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Global_Objects/RegExp))? 

I was going to see if ESLint caught that, but I guess this is a weird hybrid .handlebars file, which linting would never see.

Not sure if we need to wrap any of that in a `try..catch()` in case something implodes unexpectedly.